pkg/util/mount/mount_linux.go

```diff
@@ -243,6 +245,10 @@ func (mounter *Mounter) IsLikelyNotMountPoint(file string) (bool, error) {
 	if err != nil {
 		return true, err
 	}
+	// if file is symbolic file, skip that
+	if targetPath, err := filepath.EvalSymlinks(file); targetPath != file || err != nil {
+		return true, err
+	}
```

Comment:
how do we know we are not breaking existing functionality with this change?

Reply:
If the file is both mountpoint and symbolic link it will be break in this change. The more security method is fix in isNotMountPoint function and add explanation.